src/main/java/picard/sam/DuplicationMetrics.java

```diff
@@ -24,62 +24,84 @@
 
 package picard.sam;
 
-import htsjdk.samtools.metrics.MetricBase;
 import htsjdk.samtools.util.Histogram;
+import picard.analysis.replicates.MergeableMetricBase;
```

Comment:
Will move to `picard.analysis`.  Moving to htsjdk is a good idea, but lets get a few more uses of this class under our belt so we feel good about what it provides.